service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -200,6 +215,25 @@ public EhrExtractStatus updateEhrExtractStatusAccessDocumentDocumentReferences(
         return ehrExtractStatus;
     }
 
+    public void updateEhrExtractStatusAcknowledgment(SendAcknowledgementTaskDefinition sendAcknowledgementTaskDefinition) {
+        Query query = createQueryForConversationId(sendAcknowledgementTaskDefinition.getConversationId());
+
+        Update update = createUpdateWithUpdatedAt();
+        update.set(ACK_TASK_ID_PATH, sendAcknowledgementTaskDefinition.getTaskId());
+        update.set(ACK_MESSAGE_ID_PATH, sendAcknowledgementTaskDefinition.getMessageId());
```

Comment:
This must be the message id of the message we just sent to MHS contain the acknowledgement. The value is currently the ehrRequestMessageId.